tests/functional/test_zilien_reader.py

```diff
@@ -119,7 +130,52 @@
     ),
 }
 
+ZILIEN_COLUMNS_AMOUNT = 56
+
+
+# fmt: off
+@fixture(scope="class", params=(
```

Comment:
So a parametrized fixture? Maybe that deserves a brief comment, have never seen that before, but cool

Reply:
I did it also in my previous PR #86 here (https://github.com/ixdat/ixdat/pull/86/files#r908083418).

Reply:
Yeah, it is great. I think you also commented on it below, so probably just ignore this

Reply:
This also threw me off. I could use a comment though on what is in this particular fixture.
Maybe mention that it is `(path_to_tsv, paths_to_mpts, time_offsets)` and why that is a useful thing to use in tests.

Reply:
I described it more in the doc string and changed the variable name, where the parameters are unpacked, from `mpt_offsets` to `mpt_time_offsets`.